sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/tabular/automl_tabular.py

```diff
@@ -301,6 +303,14 @@ def set_limits(
 
             * Equal to 1, the default.
         :paramtype max_cores_per_trial: typing.Optional[int]
+        :keyword max_nodes: [Experimental] The maximum number of nodes to use for this experiment.
+
+            * For forecasting, each model is trained using max(2, int(max_nodes / max_concurrent_trials)) nodes.
+
+            * For classification/regression, each model is trained using n nodes.
```

Comment:
Is `n` total available compute nodes?

Reply:
It should me max_nodes here. I have updated it. Thanks!